test/test_parse_generic_metadata.py

```diff
@@ -480,6 +483,93 @@ async def test_rows_with_invalid_participant_meta(self, mock_graphql_query):
             )
         return
 
+    @run_as_sync
+    @patch('metamist.parser.generic_parser.query_async')
+    async def test_lrs_rows_with_arbitrary_assay_meta_columns(
+        self,
+        mock_graphql_query
+    ):
+        """
+        Test importing a single row of long read sequencing data with arbitrary assay metadata columns
+        """
+        mock_graphql_query.side_effect = self.run_graphql_query_async
+
+        rows = [
+            'Individual ID\tSample ID\tFilename\tassay_meta1\tassay_meta2',
+            'Athena\tsample_id003\t"sample_id003.filename-R1.fastq.gz,sample_id003.filename-R2.fastq.gz"\tTrue\tsome_value',
+        ]
```

Comment:
Add another row here, so you have multiple sequencing pairs (multiple assays) for a single sample, and change the meta values to check they are getting applied correctly.

One of the other style of tests you can do to test that collapse functionality:

| participant | sample | filename             | assay_meta_1   |
|-------------|--------|----------------------|----------------|
| p1          | s1     | p1-s1-01_R1.fastq.gz | assay1_forward |
| p1          | s1     | p1-s1-01_R2.fastq.gz | assay1_back    |
| p1          | s1     | p1-s1-02_R1.fastq.gz | Value1 |
| p1          | s1     | p1-s1-02_R2.fastq.gz | Value1    |